src/class-parsely.php

```diff
@@ -653,4 +670,93 @@ private function are_credentials_managed(): bool {
 
 		return true;
 	}
+
+	/**
+	 * Sets the values of managed options.
+	 *
+	 * @since 3.9.0
+	 * @access private
+	 */
+	private function set_managed_options(): void {
+		$managed_options = apply_filters( 'wp_parsely_managed_options', false );
+
+		if ( ! is_array( $managed_options ) ) {
+			return;
+		}
+
+		// Don't allow certain options to be set as managed.
+		unset(
+			$managed_options['apikey'],
+			$managed_options['api_secret'],
+			$managed_options['metadata_secret'],
+			$managed_options['track_post_types'],
+			$managed_options['track_page_types'],
+			$managed_options['plugin_version']
+		);
```

Comment:
Just a thought .... Instead of blacklisting I think we should follow whitelisting approach here i.e. only define the values that we want to be managed.

Reply:
Both approaches would work, I think this is largely opinion-based. I prefer the blocklist because:
- There are less items to add to the list.
- An allowlist would need to be updated every time a new option is added. Admittedly, not something that happens often, but would introduce a friction point and a "oh I forgot" potential.